src/Utils/badgeConnexion.js

```diff
@@ -64,7 +64,7 @@ class BadgeConnexion extends Component {
 
     {
 
-      buttons.map(button => <Button size="lg" onClick={() => {this.updatePin(this.state.ActualPin, button)}}>{ button }</Button>)
+      buttons.map(button => <Button size="lg" onMouseDown={() => {this.updatePin(this.state.ActualPin, button)}}>{ button }</Button>)
```

Comment:
Pourquoi `onMouseDown` plutôt que `onClick` ?

Reply:
pour les caisses ! ca fait des minis drag and drops sinon et du coup quand on a les doigts mouillés par les bieres ca marche pas